sotodlib/io/imprinter.py

```diff
@@ -257,7 +257,7 @@ def register_books(self, bids, obs_lists, commit=True, session=None):
             self.register_book(session, bid, obs_list, commit=False)
         if commit: session.commit()
 
-    def bind_book(self, book, session=None, output_root="out", message=""):
+    def bind_book(self, book, session=None, output_root="out", message="", hwp_root=None):
```

Comment:
I think I might prefer to no change to the function signature here because they need to be propagate downstream. Shouldn't hwp_root be already specified in the g3tsmurf config file which is stored in `self.sources[book.tel_tube]['g3tsmurf']`? This will be easier for downstream tasks.